CIE-5497 immutable CR fields email and company name (#3865)
SoftwareAG/c8y-docs: content/edge-kubernetes/manage-edge-bundle/modify-edge.md

### Basic post-installation configuration {#basic-post-installation-configuration}

> Immediately after installing Edge using the `c8yedge` tool, you will probably want to configure an SSL certificate for your Edge, and give it a custom domain. And also set the Edge tenant's name (Company name) and the admin user email.

Reviewer: ```suggestion
After installing Edge using the `c8yedge` tool, you might want to configure an SSL certificate for your Edge, give it a custom domain, and set the Edge tenant's name (company name) and the admin user email.
```

> Immediately after installing Edge using the `c8yedge` tool, you will probably want to configure an SSL certificate for your Edge, and give it a custom domain. And also set the Edge tenant's name (Company name) and the admin user email.

Reviewer: Probably you did not accept my suggestions for good reasons. But at least the last sentence shouldn't start with "And". I would remove it.

Author: Sorry, I didn't realize that I haven't accepted your suggestion. I might have added your suggestion to a batch and didn't commit it. Committed your suggestion now :)

  --key=./certs/tls.key
```

> Then apply the below configuration for configuring and SSL certificate and a custom domain:

Reviewer: ```suggestion
Then apply the below configuration for configuring an SSL certificate and a custom domain:
```
